workflows/ampliseq.nf

```diff
@@ -647,7 +649,7 @@ workflow AMPLISEQ {
             ch_multiqc_files = ch_multiqc_files.mix(CUTADAPT_WORKFLOW.out.logs.collect{it[1]}.ifEmpty([]))
         }
 
-        ch_multiqc_configs = Channel.from(ch_multiqc_config).mix(ch_multiqc_custom_config).ifEmpty([])
+        ch_multiqc_config = ch_multiqc_config.mix(ch_multiqc_custom_config).ifEmpty([])
 
```

Comment:
I deleted the line locally and tested with `NXF_VER=23.02.0-edge` & `NXF_VER=22.10.1` and it worked fine, so I suggest to delete the line.

```suggestion
```

Reply:
Fine. I suppose @jtangrot should commit...

Reply:
If you are fine with deleting that line, I am :) By the way, I don't think it was really used before, the channel was previously named ch_multiqc_configs (with an 's' at the end) and I cannot find it anywhere else. However, I thought that was a typo, and it runs nicely locally.

Reply:
Sorry, I wanted to save some time and commit already. But now github tests are queuing anyway, so no time benefit ;(

Reply:
> ch_multiqc_configs (with an 's' at the end)

indeed, and I thought it was duplicated, but no, it was even non-functional...